### PR TITLE
New package: SwayNotificationCenter-0.7.2

### PR DESCRIPTION
**Pull request by akhiljalagam**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

**Comment by akhiljalagam** [comment]
https://github.com/void-linux/void-packages/pull/40099

File: srcpkgs/swaync/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'swaync'
+pkgname=swaync
```
Comment (classabbyamp) [inline srcpkgs/swaync/template:2]: follow the upstream naming (SwayNotificationCenter)

Comment (akhiljalagam) [reply srcpkgs/swaync/template:2]: followed this actually. https://aur.archlinux.org/packages/swaync  
author used swaync too

File: srcpkgs/swaync/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+version=0.7.2
+revision=1
+wrksrc=SwayNotificationCenter-${version}
+build_style=meson
+hostmakedepends="vala glib-devel pkg-config"
+makedepends="gobject-introspection gtk+3-devel gtk-layer-shell-devel
+json-glib-devel libhandy1-devel scdoc"
```
Comment (classabbyamp) [inline srcpkgs/swaync/template:9]: scdoc shouuld definitely be hostmakedepend

File: srcpkgs/swaync/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/swaync/template:2]: we aren't the AUR. see [here](https://github.com/void-linux/void-packages/blob/master/Manual.md#programs)

Comment (akhiljalagam) [reply srcpkgs/swaync/template:2]: author used swanync only in readme file.  
https://github.com/ErikReider/SwayNotificationCenter#sway-usage

File: srcpkgs/swaync/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+depends="dbus"
+short_desc="Simple notification daemon with a GTK gui"
+maintainer="Akhil Jalagam <akhil.j@gmail.com>"
+license="GPL-3.0-or-later"
```
Comment (cinerea0) [inline srcpkgs/swaync/template:13]: Reasonably sure this should be "GPL-3.0-only"; I searched through the source tree for "or later" and didn't find any matches.

Comment (classabbyamp) [reply srcpkgs/swaync/template:13]: what about "or any later"? that's the usual text suggested by the licence

Comment (cinerea0) [reply srcpkgs/swaync/template:13]: Only in the LICENSE file when it's explaining what "or any later version" means.

Comment (akhiljalagam) [reply srcpkgs/swaync/template:13]: image
I will use GPL-3.0+

File: srcpkgs/swaync/template (hunk 1 above)
Comment (akhiljalagam) [reply srcpkgs/swaync/template]: Updated repo name. thanks.

File: srcpkgs/swaync/template (hunk 2 above)
Comment (akhiljalagam) [reply srcpkgs/swaync/template]: updated.

File: srcpkgs/swaync/template (hunk 3 above)
Comment (akhiljalagam) [reply srcpkgs/swaync/template]: using 'GPL-3.0-only' now.

**Comment by akhiljalagam** [comment]
Why is this package still not merged into the main repo?

**Comment by akhiljalagam** [comment]
> Why is this package still not merged into the main repo?

:hourglass:

**Comment by akhiljalagam** [comment]
<meta http-equiv="content-type" content="text/html; charset=utf-8">

> Why is this package still not merged into the main repo?

**Comment by hochata** [comment]
Hi,
are there any blockers for this MR?

**Comment by hholst80** [comment]
In other distros this package is called "swaync" and given how hard it is to find packages I think it would make sense with at least some kind of virtual link / pkg to associate the name "swaync" with the pkg "SwayNotificatonCenter"?

**Comment by akhiljalagam** [comment]
> In other distros this package is called "swaync" and given how hard it is to find packages I think it would make sense with at least some kind of virtual link / pkg to associate the name "swaync" with the pkg "SwayNotificatonCenter"?

Furthermore, including "SwayNotificationCenter" in the package's short description will assist users in locating it conveniently through the `xbps-search` command.

**Pull request closed without merging**